inference/core/registries/roboflow.py

```diff
@@ -21,6 +21,7 @@
     "sam": ("embed", "sam"),
     "gaze": ("gaze", "l2cs"),
     "doctr": ("ocr", "doctr"),
+    "grounding_dino": ("object-detection", "grounding_dino"),
```

Comment:
Should we use a `-` instead of `_` in the string `grounding_dino`? This just caught my eye as odd with a `-` in `object-detection` but then a `_`.